### DISCUSSION TITLE
FirebaseInAppMessaging SPM naming convention

### DISCUSSION THREAD

**Discussion by bartosz-pawelczyk** [firebase/firebase-ios-sdk#12278] [General]
Hi, 
Release notes contains information about migration from `FirebaseInAppMessagingSwift` to `FirebaseInAppMessaging`.

<img width="884" alt="Screenshot 2024-01-16 at 13 19 09" src="https://github.com/firebase/firebase-ios-sdk/assets/90895410/d3f9d67e-fb20-474d-8adf-035ea74a843e">

However `Package.swift` (10.19.1) contains old naming with `Beta` postfix and xcode cannot find as `FirebaseInAppMessaging`. Was it done on purpose?

```
    .library(
      name: "FirebaseInAppMessaging-Beta",
      targets: ["FirebaseInAppMessagingTarget"]
    ),
    .library(
      name: "FirebaseInAppMessagingSwift-Beta",
      targets: ["FirebaseInAppMessagingSwift"]
    ),
```

**Comment by ncooke3** [comment]
Hi @bartosz-pawelczyk, sorry for the confusion. This was not done on purpose. The release notes probably should have referred to the modules with the postfix to make that clear. The consolidation of the Swift extension into Firebase In App Messaging had no effect on In App Messaging being in beta.
